### PR TITLE
Remove dependency of `Y2Users::HelpTexts` on `UsersSimple`

### PR DESCRIPTION
**Pull request by dgdavid**
### Problem

`Y2Users::HelpTexts` depends on `Yast::UsersSimple` just for the `Yast::UsersSimple.ValidPasswordHelptext`, which simply returns a localized string.

https://trello.com/c/j0t4rJAZ/2430-1-remove-dependency-of-y2usershelptexts-on-userssimple

### Solution

As we agreed, for this special case the simpler solution is just to duplicate the localized string in `Y2Users::HelpText`.

### Tests

Tested manually checking that the translation still working as expected.

passwords_help_text

### CODE REVIEW

**Review by ancorgs — APPROVED** [review]
LGTM. But maybe we should add a comment in both `#valid_password_text` and `#ValidPPasswordHelpText` mentioning the existence of each other.

File: src/lib/y2users/help_texts.rb [hunk 1]
```diff
@@ -47,7 +46,15 @@ def validation_config
     #
     # @return [String] formatted and localized text
     def valid_password_text
-      Yast::UsersSimple.ValidPasswordHelptext
+      # NOTE: to avoid dependencies, this help text is a duplicate of
+      #   Yast::UsersSimple.ValidPPasswordHelpText. Please, keep them in sync.
```
Comment (ancorgs) [inline src/lib/y2users/help_texts.rb:50]: ValidPPassword, double "P"

Comment (dgdavid) [reply src/lib/y2users/help_texts.rb:50]: :see_no_evil: I'll hide your comment right away! It never happened :stuck_out_tongue_winking_eye:

**Review by ancorgs — APPROVED** [review]
LGTM

**Review by jreidinger — APPROVED** [review]

**Pull request merged**